includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -33,6 +33,24 @@ public function getHref( $name, $lang ) {
 		return WikiFactory::getLocalEnvURL( $href );
 	}
 
+	/**
+	 * @param $name string key for href
+	 * @param $lang string two letter language code
+	 * @param $cityId int
+	 * @return string full URL, in case of lang specific URL missing, default one is returned
+	 */
+	public function getLocalHref( $name, $lang, $cityId) {
+		$lang = $this->getLangWithFallback( $lang );
+
+		$href = $this->hrefs[$lang][$name] ?? $this->hrefs['default'][$name];
+
+		return WikiFactory::getLocalEnvURL(
+			WikiFactory::cityUrlToLanguagePath(
```

Comment:
Since we're now using this in a few places (and may start using this in more), I wonder if it makes sense to have a helper method in WikiFactory to get this for an ID, what do you think?